ts/parts-more/Polar.ts

```diff
@@ -1009,7 +1104,9 @@ H.SVGRenderer.prototype.clipCircle = function (
             id: id
         }).add(this.defs);
 
-    wrapper = this.circle(x, y, r).add(clipPath);
+    wrapper = innerR ?
+        this.arc(x, y, r, innerR, 0, 2 * Math.PI).add(clipPath) :
+        wrapper = this.circle(x, y, r).add(clipPath);
```

Comment:
The `wrapper = ` part shouldn't be necessary in this ternary?

Reply:
Yes, sorry missed that. Thank you!

Reply:
Corrected.